src/Microsoft.Health.Fhir.Shared.Api/Features/ContentTypes/ContentTypeService.cs

```diff
@@ -82,6 +83,13 @@ public async Task CheckRequestedContentTypeAsync(HttpContext httpContext)
                     }
                 }
             }
+
+            string prettyParameterValue = GetPrettyFromQueryString(httpContext);
+
+            if (prettyParameterValue != null && !bool.TryParse(prettyParameterValue, out bool isPretty))
```

Comment:
You might be able to just call `bool.TryParse` without the null check. If memory serves, it does a null check internally. Also, we can probably change the out parameter to be `out _` since we don't actually care about the return value.

Reply:
Right, I originally didn't have this null check for that reason.  What I realized was here `prettyParameterValue=null` is used to indicate that `_pretty` wasn't included in the query string (and we wouldn't want this to fail in that case).

And cool thanks, didn't know you could discard out variables. Fixed in 19651a.